napi.h

```diff
@@ -355,6 +357,20 @@ namespace Napi {
     /// be needed to store this BigInt (i.e. the return value of `WordCount()`).
     void ToWords(int* sign_bit, size_t* word_count, uint64_t* words);
   };
+
+  /// A JavaScript date value.
+  class Date : public Value {
+  public:
+    static Date New(
```

Comment:
Please add this comment here: ` /// Creates a new Date value from a double primitive.`